### PR TITLE
update db with output_id and reviewed

### PR DESCRIPTION
**Pull request by brianyu28**
I've deleted the old initial migration and replaced it with this one, which has the following changes:

- `reviewed` field has been added to `inputs`, with default value `False`.
- `input_id` has been removed from `outputs`, replaced by `output_id` in `inputs`, which is nullable.

Per your Asana comment, probably best to get rid of the current server-side tables before merging, such that the new initial migration can replace them.

I believe @annieechen is working on logging the inputs/outputs.

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -16,19 +16,20 @@
...
     script = db.Column(db.Text, nullable=False)
     username = db.Column(db.String(length=32), nullable=True, index=True)
     created = db.Column(db.DateTime, default=func.now(), index=True)
-
-class Output(db.Model):
-    __tablename__ = "outputs"
-    id = db.Column(db.BigInteger, primary_key=True, nullable=False, autoincrement=True)
-    input_id = db.Column(db.BigInteger, db.ForeignKey("inputs.id"))
-    output = db.Column(db.Text)
+    output_id = db.Column(db.BigInteger, db.ForeignKey("outputs.id"), nullable=True)
```
Comment (dmalan) [inline manage.py]: Does declaring this a foreign key automatically create its index?

File: manage.py [hunk 2]
```diff
@@ -16,19 +16,20 @@
...
+    reviewed = db.Column(db.Boolean, default=False)
```
Comment (dmalan) [inline manage.py]: This should be indexed too so we can search on it more efficiently.

File: migrations/versions/3b25e13c81bf_.py [hunk 3]
```diff
@@ -16,30 +16,31 @@
...
-    sa.Column('input_id', sa.BigInteger(), nullable=True),
-    sa.Column('output', sa.Text(), nullable=True),
-    sa.ForeignKeyConstraint(['input_id'], ['inputs.id'], ),
-    sa.PrimaryKeyConstraint('id')
-    )
     ### end Alembic commands ###
 
 
 def downgrade():
     ### commands auto generated by Alembic - please adjust! ###
```
Comment (dmalan) [inline migrations/versions/3b25e13c81bf_.py]: Comment outdated.

File: migrations/versions/3b25e13c81bf_.py [hunk 4]
```diff
@@ -16,30 +16,31 @@
 
 def upgrade():
     ### commands auto generated by Alembic - please adjust! ###
```
Comment (dmalan) [inline migrations/versions/3b25e13c81bf_.py]: Comment outdated.

File: manage.py (hunk 1 above)
Comment (brianyu28) [inline manage.py]: Yeah, InnoDB with MySQL requires that all foreign keys be indexed.

**Comment by brianyu28** [comment]
Made adjustments in 03079c8

**Pull request merged**